packages/app/components/Queue/Student/StudentQuestionRephraseModal.tsx

```diff
@@ -0,0 +1,36 @@
+import { RephraseQuestionPayload } from "@koh/common";
+import { Button } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { ReactElement } from "react";
+
+type StudentQuestionRephraseModalProps = {
+  courseId: number;
+  payload: RephraseQuestionPayload;
+  handleClose: (number) => void;
+};
+export default function StudentQuestionRephraseModal({
+  courseId,
+  payload,
+  handleClose,
+}: StudentQuestionRephraseModalProps): ReactElement {
+  return (
+    <Modal
+      visible={true}
+      closable={false}
+      footer={[
+        <Button
+          type={"primary"}
+          key={"continue"}
+          href={`/course/${courseId}/queue/${payload.queueId}?edit_question=true`}
+          onClick={handleClose}
```

Comment:
Something isn't working with `handleClose`, does onClick behave weirdly with async functions? Or is there a problem with the API request?